ts/features/wallet/onboarding/satispay/screens/add/AddSatispayScreen.tsx

```diff
@@ -29,20 +32,36 @@ import LoadAddSatispayComponent from "./LoadAddSatispayComponent";
 type Props = ReturnType<typeof mapDispatchToProps> &
   ReturnType<typeof mapStateToProps>;
 
-const DisplayFoundSatispay = (props: Props) => (
-  <SafeAreaView style={IOStyles.flex}>
-    <View style={IOStyles.flex}>
-      <H1>Satispay Account found</H1>
-    </View>
-    <FooterWithButtons
-      type={"TwoButtonsInlineThird"}
-      leftButton={cancelButtonProps(props.cancel)}
-      rightButton={confirmButtonProps(
-        () => props.satispay && props.confirm(props.satispay)
-      )}
-    />
-  </SafeAreaView>
-);
+const loadLocales = () => ({
+  headerTitle: I18n.t("wallet.onboarding.satispay.headerTitle"),
+  title: I18n.t("wallet.onboarding.satispay.add.title")
+});
+
+const DisplayFoundSatispay = (props: Props) => {
+  const { headerTitle, title } = loadLocales();
+  return (
+    <BaseScreenComponent goBack={true} headerTitle={headerTitle}>
+      <SafeAreaView style={IOStyles.flex}>
+        <ScrollView>
+          <View style={IOStyles.horizontalContentPadding}>
+            <View spacer={true} />
+            <H1>{title}</H1>
+            <View spacer={true} extralarge={true} />
+            <SatispayCard />
+          </View>
+        </ScrollView>
+        <FooterWithButtons
+          type={"TwoButtonsInlineThird"}
+          leftButton={cancelButtonProps(props.cancel)}
+          rightButton={confirmButtonProps(
+            () => props.satispay && props.confirm(props.satispay),
+            I18n.t("global.buttons.continue")
+          )}
+        />
+      </SafeAreaView>
+    </BaseScreenComponent>
+  );
+};
 
```

Comment:
I guess we are not handling the error case while app is adding satispay to the wallets
How to reproduce:
- search for satispay 
- show the satispay info screen
- stop local server (or return an error from the relative API)
- add satispay

It displays always the loading state

Maybe `<LoadAddSatispayComponent />` should listen about wallet insertion errors
`hasError: isError(onboardingSatispayAddingResultSelector(state))`